### ISSUE TITLE
Add FTP upload capabilities!

### ISSUE THREAD

**Issue by rickmed** [matjaztrontelj/PinegrowCommunity#23]
Awesome piece of software

**Comment by matjaztrontelj** [comment]
My thinking is that since Pinegrow is a desktop app it is simple to use
external FTP program (or GIT, Dropbox,...) to upload changed files to the
server. Some FTP clients also let you define watchers that automatically
upload changed files.

Please let me know why you feel that build-in FTP support would add
substantial value to Pinegrow.

On Sun, Mar 30, 2014 at 10:03 AM, ricmed205 notifications@github.comwrote:

> —
> Reply to this email directly or view it on GitHubhttps://github.com/matjaztrontelj/PinegrowCommunity/issues/23
> .

## 

_Matjaz Trontelj_
Co-founder of Pinegrow http://pinegrow.com/, Booklershttp://getbooklers.com
 and pllop.it

Our new product Pinegrow is out. Kindly take a look: www.pinegrow.com
Please check out our new apps Life is
Beautifulhttps://itunes.apple.com/us/app/life-is-beautiful-create-posters/id607999197?mt=8
, Bible Surprisehttps://itunes.apple.com/us/app/bible-surprise-your-photos/id643673873?mt=8,
Pope Francishttps://itunes.apple.com/us/app/pope-francis-daily-surprise/id685158626?mt=8
, Zen Habits https://itunes.apple.com/th/app/id681123869?mt=8

**Comment by rickmed** [comment]
You're right. In that sense it's not paramount. Thanks

**Comment by bsusala** [comment]
I would very welcome a FTP tool integrated! It would substantially add value. Why not complete development steps up to the final stage? Although perfectible, Dreamweaver's site manager is a good starting point. Other examples exists, I am sure. Pinegrow might deserve the features of an fully integrated development tool :)

**Comment by tmikaeld** [comment]
I believe this would be servered better as a plugin and not a core-feature.

**Comment by bsusala** [comment]
plugin or not, okay, you decide, but something integrated in the interface.

**Comment by niutech** [comment]
I don't think it's necessary. FTP/SVN/Git/WebDAV support should be at the filesystem level, not application level. There are a lot of existing solutions for this, e.g. FUSE, mapped FTP & network drives.

**Comment by narration-sd** [comment]
I agree that though this ability is strongly needed, it's much better to let it be provided from outside. I presently use Cyberduck, which works very easily, can do automatic opens and saves. Expandrive also looks good.

You have a product that can do very useful things, Matjaz, from 1.1 -- and that video was very good to express it, thanks. I would much rather see first-class templating support, SASS, Foundation, etc. as valuable uses of your time.

For idea on how useful templating support could be, have a look at Craft, a great cms that implements Twig, at buildwithcraft.com. Note that you can download a very capable level free, and that this installs (and updates) almost magically, including on your laptop, where it will run fine. I think you'll appreciate the design, anyway. They are very good people there (and one in Riga ;) )

A very nice job all around, the two of you. I'm reminded of a Slovenian I worked with in Basel.
Clive

p.s. I'm already trying 1.1, and immediately impressed with how it downloaded a complex webpage. The most important key in having this ability is that I can really see inside a video player I didn't write, and deal with how it is not quite working in the way that's needed. This kind of scenario might be one more to highlight as to how Pinegrow can be useful?
